<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="Manual_files/filelist.xml">
<link rel=Preview href="Manual_files/preview.wmf">
<link rel=Edit-Time-Data href="Manual_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title> JyMet user manual</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>John</o:Author>
  <o:Template>Normal</o:Template>
  <o:LastAuthor>Huihai</o:LastAuthor>
  <o:Revision>139</o:Revision>
  <o:TotalTime>3277</o:TotalTime>
  <o:Created>2012-03-02T14:19:00Z</o:Created>
  <o:LastSaved>2015-07-27T20:21:00Z</o:LastSaved>
  <o:Pages>4</o:Pages>
  <o:Words>4652</o:Words>
  <o:Characters>26518</o:Characters>
  <o:Lines>220</o:Lines>
  <o:Paragraphs>62</o:Paragraphs>
  <o:CharactersWithSpaces>31108</o:CharactersWithSpaces>
  <o:Version>14.00</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<link rel=themeData href="Manual_files/themedata.thmx">
<link rel=colorSchemeMapping href="Manual_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>130</w:Zoom>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-GB</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:DontVertAlignCellWithSp/>
   <w:DontBreakConstrainedForcedTables/>
   <w:DontVertAlignInTxbx/>
   <w:Word11KerningPairs/>
   <w:CachedColBalance/>
   <w:UseFELayout/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="267">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--span.OVERACC2
	{left:.8em;
	position:relative;
	top:-1.2ex;}
span.OVERACC1
	{left:.6em;
	position:relative;
	top:-1.2ex;}

 /* Font Definitions */
 @font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-alt:SimSun;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 680460288 22 0 262145 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-alt:SimSun;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 680460288 22 0 262145 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1073786111 1 0 415 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-520081665 -1073717157 41 0 66047 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 680460288 22 0 262145 0;}
@font-face
	{font-family:"DejaVu Sans";
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:auto;
	mso-font-signature:0 0 0 0 0 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	mso-outline-level:1;
	font-size:24.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
h2
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 2 Char";
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	mso-outline-level:2;
	font-size:18.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Header Char";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 207.65pt right 415.3pt;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Footer Char";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 207.65pt right 415.3pt;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
a:link, span.MsoHyperlink
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	text-decoration:underline;
	text-underline:single;}
tt
	{mso-style-noshow:yes;
	mso-style-priority:99;
	font-family:"Courier New";
	mso-ascii-font-family:"Courier New";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:"Courier New";
	mso-bidi-font-family:"Courier New";}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:14.0pt;
	font-family:"Cambria","serif";
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#365F91;
	mso-themecolor:accent1;
	mso-themeshade:191;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 2";
	mso-ansi-font-size:13.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:"Cambria","serif";
	mso-ascii-font-family:Cambria;
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:Cambria;
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#4F81BD;
	mso-themecolor:accent1;
	font-weight:bold;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-ascii-font-family:Tahoma;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;}
span.roman
	{mso-style-name:roman;
	mso-style-unhide:no;
	font-family:"Times New Roman","serif";
	mso-ascii-font-family:"Times New Roman";
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	font-weight:normal;
	font-style:normal;}
span.st
	{mso-style-name:st;
	mso-style-unhide:no;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:10.0pt;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	mso-fareast-font-family:SimSun;}
 /* Page Definitions */
 @page
	{mso-footnote-separator:url("Manual_files/header.html") fs;
	mso-footnote-continuation-separator:url("Manual_files/header.html") fcs;
	mso-endnote-separator:url("Manual_files/header.html") es;
	mso-endnote-continuation-separator:url("Manual_files/header.html") ecs;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:72.0pt 90.0pt 72.0pt 90.0pt;
	mso-header-margin:35.4pt;
	mso-footer-margin:35.4pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:100876420;
	mso-list-template-ids:2084102698;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l0:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1
	{mso-list-id:258100464;
	mso-list-template-ids:-737086330;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l1:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2
	{mso-list-id:377512149;
	mso-list-template-ids:292336442;}
@list l2:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l2:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3
	{mso-list-id:598876629;
	mso-list-template-ids:811910988;}
@list l3:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l3:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4
	{mso-list-id:747507570;
	mso-list-template-ids:2075020756;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l4:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5
	{mso-list-id:836387175;
	mso-list-template-ids:-2114275262;}
@list l5:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l5:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l5:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6
	{mso-list-id:884872201;
	mso-list-template-ids:-1832113162;}
@list l6:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l6:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l6:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="2049"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-GB link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=WordSection1>

<h1 align=center style='text-align:center'><span style='mso-fareast-font-family:
"Times New Roman"'>JyMet user manual <o:p></o:p></span></h1>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>JyMet
(version 2.3</span><span style='mso-fareast-font-family:SimSun'>4</span><span
style='mso-fareast-font-family:"Times New Roman"'>) is a graphical user
interface to the program SurreyFBA, which is a command-line tool for metabolic
model analysis. JyMet enables creating, editing, saving and loading metabolic
models, definition of modelling problems in a simple scripting format and their
solution using SurreyFBA, as well as visualisation of models and results. This
manual is about JyMet only, more details about SurreyFBA can be found in its manual
(further referred as SFM). <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<h2><a name="tth_sEc1"><span style='mso-fareast-font-family:"Times New Roman"'>1</span></a><span
style='mso-fareast-font-family:"Times New Roman"'>&nbsp;&nbsp;Model file
structure<o:p></o:p></span></h2>

<p class=MsoNormal><a name="sec_struct"></a><span style='mso-fareast-font-family:
"Times New Roman"'>A JyMet model consists of four hierarchical levels: genes,
enzymes, reactions and metabolites. An element of each level can be associated
with elements on the higher level, called parents (e.g. genes encoding an
enzyme, enzymes catalysing a reaction, or reactions using (i.e. consuming or
producing) a metabolite, and on the lower level, called children. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>Models
can be represented in one of the following tab-delimited formats: <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l6 level1 lfo1;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>SBML format: a widely used model representation format
     based on XML, which defines model into classified components, currently
     only Level 2 Version 1 being supported. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l6 level1 lfo1;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>`simple' format: which describes only reactions and is
     also accepted by SurreyFBA (see SFM). File can be in text format or Excel
     ‘xls’ workbook format. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l6 level1 lfo1;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>`extended' format: which consists of four parts,
     describing the hierarchical levels of the model. File can be in text
     format or Excel ‘xls’ workbook format. <o:p></o:p></span></li>
</ul>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>Each
part starts with the name of the corresponding hierarchical level, followed by
lines describing individual elements. The description of each element consists
of tab-delimited values of its attributes, listed below: <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Genes</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: ID, Name, Children, Predictions, Comment. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Enzymes</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: ID, Name, Children, Rule, Compartments, Comment. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Reactions</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: ID, Name, Equation, LB, UB, Rule, Comment. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Metabolites</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: ID, Name, External, Canonical SMILES, Isomeric
     SMILES, Molecular mass, Comment. <o:p></o:p></span></li>
</ul>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>The
meaning of the attributes is shown below: <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>ID: a modelwide unique identifier <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Name: arbitrary string, may be non-unique <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Children: list of the children’s IDs. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Predictions: list of BLAST predictions. Each prediction
     consists of an EC number followed by a colon and the e-value. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Rule: a logical rule representing the association with
     the parents; may include IDs, AND and OR operators (case insensitive), and
     parentheses, all space-delimited. E.g. the following rule: `g1 and ( g2 or
     g3 )' means that the given enzyme is encoded by the genes g1 and either g2
     or g3. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Equation: a reaction equation, consisting of metabolite
     identifiers, stoichiometric coefficients (the coefficient 1 can be
     skipped), `+' and `=' operators, all space-delimited. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>LB: a floating point number defining the lower flux
     rate bound. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>UB: a floating point number defining the upper flux
     rate bound. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>External: Boolean value `True' if the metabolite is
     external and `False', otherwise. If you set the externals by specifying a
     tag at <b style='mso-bidi-font-weight:normal'>Solve &gt; Externality tag</b>,
     the externals specified by Boolean value or problem table will be
     overwritten. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Compartment, Canonical SMILES, Isomeric SMILES,
     Molecular mass: arbitrary strings. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo3;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Comment: arbitrary string starting with the hash
     symbol. <o:p></o:p></span></li>
</ul>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>Note
that a reaction equation may be empty (i.e. consist of a `=' operator only) and
a metabolite may be used by no reactions. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<h2><a name="tth_sEc2"><span style='mso-fareast-font-family:"Times New Roman"'>2</span></a><span
style='mso-fareast-font-family:"Times New Roman"'>&nbsp;&nbsp;Model editor<o:p></o:p></span></h2>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>JyMet
can be used for work with multiple models simultaneously. Each model is shown
in a separate tab, whose title shows its name (if it is newly created) or the
complete path (if it is loaded from a file). A model tab consists of five
lower-level tabs, titled `Genes', `Enzymes', 'Reactions', `Metabolites' and
`Problem'. Each tab contains an empty table (spreadsheet); the first four
describe the corresponding hierarchical levels of the model, with rows
corresponding to the elements and columns corresponding to the attributes, as
described in Section&nbsp;<a href="#sec_struct">1</a>. Note that the
`Metabolites' table contains an additional column `Used by', which shows the
reactions using the given metabolites (it is filled in accordance with the
equations). The `Problem' table is described in Section&nbsp;<a href="#sec_prob">3</a>.
<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Creating
a model&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>Select
<b>File &gt; New model</b> in the menu and enter a model name into the field in
the dialog window. A new model tab with empty tables will appear. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Adding
elements&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Select the table showing the hierarchical layer to which new elements are to be
added; then select <b>Edit &gt; Insert rows</b> and enter the space separated
names of the new elements into the appearing dialog box. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Deleting
elements&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Select the rows (i.e. any cells in those rows) describing the elements to be
deleted, then select <b>Edit &gt; Delete rows</b>. Note that a gene or an enzyme
cannot be deleted if it has children, so the &quot;Rule&quot; attribute of the
corresponding children has to be edited first (see below). <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Editing
elements&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Attributes are edited by simply entering new values into the corresponding
cells. Note that the columns `Children' and `Used by' are not editable. If a
new value of a `Rule' or `Equation' attribute includes new IDs, new elements
with these IDs are included into the model and the corresponding rows appear in
tables. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Searching
and filtering table contents&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:
"Times New Roman"'> Select <b>Edit &gt; Search</b> and enter a regular
expression into the appearing dialog box to search for it in the table
currently selected. The table will be filtered to the rows containing matches
to this expression. To limit the search to a subset of columns, select them
before the search. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>Select
<b>View &gt; Show &gt; Children</b> or <b>View &gt; Show &gt; Parents</b> to
filter the adjacent table to the rows describing the children or the parents of
the elements currently selected, respectively. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>Select
<b>View &gt; Show &gt; All</b> to unfilter a table. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Opening
a model&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Select <b>File &gt; Open model</b> and select a file in one of the formats
described in Section&nbsp;<a href="#sec_struct">1</a>. A new model tab with
tables populated with the model contents will appear. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Saving
a model&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Select <b>File &gt; Save model</b> and specify a file name. The model will be
saved in the `extended' format (see Section&nbsp;<a href="#sec_struct">1</a>). <o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-fareast-font-family:"Times New Roman"'>Importing a SBML model </span></b><span
style='mso-fareast-font-family:"Times New Roman"'><span
style='mso-spacerun:yes'> </span>Select <b>File &gt; </b><b style='mso-bidi-font-weight:
normal'>Import SBML</b> and select a file name (xml file).<span
style='mso-spacerun:yes'>  </span>The SBML file will be parsed and loaded and
model tabs will appear. <o:p></o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span
style='mso-fareast-font-family:"Times New Roman"'>Exporting a SBML model</span></b><span
style='mso-fareast-font-family:"Times New Roman"'><span
style='mso-spacerun:yes'>  </span>Select <b>File &gt; Export SBML</b> and
specify a xml file name. The model will be converted into SMBL model format,
where making sure externality tag is specified (in <b style='mso-bidi-font-weight:
normal'>Solve</b> &gt; <b style='mso-bidi-font-weight:normal'>Externality tag</b>)
if the externality tag is not ‘_b’.</span><span style='mso-fareast-font-family:
SimSun'> To conform to the Sid type for SBML components, apart from&nbsp;
characters of letters, numbers and </span><span class=st>underscores, </span><span
style='mso-fareast-font-family:SimSun'>some special characters are replace with
'_'+ Abbreviation + '_', for example: ',' replaced with '_</span> <span
style='mso-fareast-font-family:SimSun'>COMMA_'. Because Model name is file
name, the name should be in Sid type.<o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Saving
a table&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Select <b>File &gt; Save table</b> and select a file name. The currently
selected low-level tab will be saved in a file, whose format depends on the
contents of the tab (tab-delimited for tables and graphical files for
diagrams). If a `Reactions' table is selected, the resulting file will be in
the `simple' model format (see Section&nbsp;<a href="#sec_struct">1</a>). <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Printing&nbsp;&nbsp;</span></b><span
style='mso-fareast-font-family:"Times New Roman"'> Select <b>File &gt; Print</b>
to print the contents of the tab currently selected. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<h2><a name="tth_sEc3"><span style='mso-fareast-font-family:"Times New Roman"'>3</span></a><span
style='mso-fareast-font-family:"Times New Roman"'>&nbsp;&nbsp;Problem solver<a
name="sec_prob"> </a><o:p></o:p></span></h2>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>JyMet
uses SurreyFBA to solve various metabolic modelling problems, launching it as a
separate background process. Before a problem or a sequence of problems can be
solved, they have to be defined in the `Problem' table associated with the
model to which it is to be applied. The content of this table is then passed to
SurreyFBA as the problem file. The output of the solution is shown in a
separate table and can be visualised in a diagram or a plot. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Defining
a problem&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Each row in a `Problem' table contains one line of the problem file, whose
syntax can be found in SFM. The `Expression' column may contain the following
data: <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><tt><span style='font-size:
     10.0pt'>‘!dir: obj_fun'</span></tt><span style='mso-fareast-font-family:
     "Times New Roman"'> where dir is either &quot;min&quot; for minimisation
     or &quot;max&quot; for maximisation, and </span><tt><span
     style='font-size:10.0pt'>obj_fun</span></tt><span style='mso-fareast-font-family:
     "Times New Roman"'> is the objective function, defined as a linear
     expression including reaction or metabolite names, stoichiometric
     coefficients (the coefficient 1 can be skipped), and `+' operators, all
     space-delimited, e.g. `m1 + 0.5 r2'. Note that the `-' operators are not
     allowed, so negative stoichiometric coefficients should be used instead,
     e.g. `m1 + -0.5 r2'. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><tt><span style='font-size:
     10.0pt'>‘? arg_list'</span></tt><span style='mso-fareast-font-family:"Times New Roman"'>
     where </span><tt><span style='font-size:10.0pt'>arg_list</span></tt><span
     style='mso-fareast-font-family:"Times New Roman"'> is the argument list in
     a method-specific format (see below). <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><tt><span style='font-size:
     10.0pt'>‘\ ext_list'</span></tt><span style='mso-fareast-font-family:"Times New Roman"'>
     where </span><tt><span style='font-size:10.0pt'>ext_list</span></tt><span
     style='mso-fareast-font-family:"Times New Roman"'> is a space-delimited
     list of external metabolites, overriding the list defined in the model. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>a space-delimited linear expression (the syntax is the
     same as for objective functions). <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><tt><span style='font-size:
     10.0pt'>‘;'</span></tt><span style='mso-fareast-font-family:"Times New Roman"'>
     end of record. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>For (Fast) iMAT method, </span><tt><span
     style='font-size:10.0pt'>‘!expression_name’</span></tt><span
     style='mso-fareast-font-family:"Times New Roman"'> indicates which column
     of gene/enzyme expression profile in the expression table, only one name
     for a record. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>For GIM3E method, </span><tt><span style='font-size:
     10.0pt'>‘!objective,array’</span></tt><span style='mso-fareast-font-family:
     "Times New Roman"'> specifying an objective reaction and a metabolic array
     of metabolic signal profile, delimited by a comma.<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l5 level1 lfo4;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>For problems SGNI and WGNI, </span><tt><span
     style='font-size:10.0pt'>‘!nutrient_names’</span></tt><span
     style='mso-fareast-font-family:"Times New Roman"'> defines the space of
     growth media under which we perform prediction of GNIs, names are single
     space-delimited; and </span><tt><span style='font-size:10.0pt'>‘$nutrient
     names’</span></tt><span style='mso-fareast-font-family:"Times New Roman"'>
     designates the nutrients to be always present and rest nutrients will be
     set absent in the rest nutrients. <o:p></o:p></span></li>
</ul>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>The
objective function, the argument list, and the list of external metabolites
apply to all subsequent records until redefined. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>The
`LB' and `UB' columns can only contain the lower and upper bounds of the linear
expressions defined in the corresponding rows, thus forming linear constraints.
If the upper bound is skipped, the value is fixed to the lower bound. The
constraints apply everywhere once defined. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>The
`Comment' column contains comments. Note that comments are forbidden in all
rows except those describing linear constraints. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>To
insert new rows into the `Problem' table, select <b>Edit &gt; Insert rows</b>
and enter the number of rows into the appearing dialog box. To delete rows,
select them and select <b>Edit &gt; Delete rows</b>. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>To
save a problem, select the `Problem' table and <b>File &gt; Save table</b>; the
problem will be saved in the tab-delimited format of SurreyFBA problem files.
To open a problem file, select <b>File &gt; Open problem</b>; the contents will
appear in the `Problem' table. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Analysis
methods&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
JyMet implements a range of analysis methods, corresponding to the SurreyFBA
`program' options. The list of the methods and their specific argument list
formats is shown below: <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Objective value</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>:compute the optimal objective function value and show
     it (no args) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Objective value and status</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'>:compute the optimal
     objective function value, show it and the solution status, e.g. OPTIMAL,
     FEASIBLE, UNDEFINED etc. (no args) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>FBA</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: (Flux Balance Analysis) compute a flux distribution
     corresponding to an optimal value of the objective function (no args) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>FVA</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: (Flux Variability Analysis) compute the minimal and
     maximal flux rates of the query reactions, corresponding to the optimal
     objective function value (args: space-separated list of reactions; if not
     provided, all reactions are considered) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Elementary mode</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: compute one or more elementary flux mode involving
     the query reaction. The objective is to minimise the number of reactions
     involved (if the solver is MILP), or the sum of flux rates in the flux
     distribution (otherwise). The first argument is a reaction to be involved
     in the mode. The second argument is the number of modes to be computed (1
     by default). If the solver is MILP, an additional scaling factor between 0
     and 1 can be provided, since all flux rates in the resulting distribution
     must be in this range. The default scaling factor is 0.001; note that
     decreasing it may result in longer solution times, while increasing may
     result in the impossibility of a solution. (args: `reaction [number,
     scale_factor]'). <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Knock-out analysis</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: same as objstat, but the query reactions/enzymes are
     knocked out (args: space-separated list of reactions/enzymes) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Essential reactions</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: detect the essential reactions, by blocking which the
     value of the objective function becomes zero (args: space-separated list
     of query reactions; if not provided, all reactions are considered) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Live reactions</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: detect the live reactions, i.e. those able to carry a
     steady state flux (args: space-separated list of query reactions; if not
     provided, all reactions are considered) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Plot</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: iteratively compute the optimal objective function
     values corresponding to the incrementing/decrementing flux rates of the
     query reaction (args: `reaction from to step', e.g. `r2 0 1 0.1') <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Plot 3D</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: iteratively compute the optimal objective function
     values corresponding to the incrementing/decrementing flux rates of two
     query reactions (args: `reaction1 from to step reaction2 from to step',
     e.g. `r2 0 1 0.1 r3 -1 1 0.2') <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Minimal substrate sets</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'>: compute one or more
     minimal (i.e. irreducible) sets of external substrates which are
     sufficient for a given linear combination of external metabolites
     (supplied as the objective function) to be produced. E.g. if the objective
     function is biomass, the result would represent minimal media sufficient
     for growth. Note that some results may correspond to product compositions
     including additional by-products, not defined in the objective function
     (e.g. water and CO<sub>2</sub>). The method uses the MILP solver only. The
     first argument is the number of sets to be calculated (1 by default); the
     second argument is a scaling factor (see `Elementary modes'; args:
     `reaction [number, scale_factor']). <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Minimal product sets</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: compute one or more minimal (i.e. irreducible) sets
     of external products which can be released while a given linear
     combination of external metabolites (supplied as the objective function)
     is consumed. E.g. if the objective function is &quot;glucose + 2
     water&quot;, the result in a fermenting organism would represent minimal
     sets of fermentation products. Note that all substrates must be defined,
     since no additional co-substrates are allowed. The arguments are identical
     to those for minimal substrate sets. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Unconserved metabolites</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'>: detect the unconserved
     metabolites; e.g. those whose molecular masses cannot be simultaneously
     assigned positive values (no args) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Orphan metabolites</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: detect the orphan metabolites (i.e. those used by
     less than two reactions) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b><span style='mso-fareast-font-family:
     "Times New Roman"'>Connected components</span></b><span style='mso-fareast-font-family:
     "Times New Roman"'>: detect the weakly connected components <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>iMAT
     method:</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
     <span style='mso-bidi-font-weight:bold'>flux Activity Analysis by
     shilomi’s method which gives a qualitative prediction about flux activity
     state for reactions and genes without requiring the biomass synthesis
     reaction. Apart from genome-scale model, shlomi only needs absolute gene
     expression data which should reflect the absolute transcript abundances of
     genes under one particular growth condition. Parameter ‘<b>Threshold’</b>
     is positive threshold of active flux which is used to judge if a reaction
     is active or not. If not set, the default value is 1. (args:
     space-separated list of query reactions and genes; if not provided, all
     reactions and genes are considered).<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>Fast iMAT
     method:</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
     <span style='mso-bidi-font-weight:bold'>an alternative method to iMAT
     method which is much faster than shlomi’s method but only for reactions,
     the method relies on MILP. (args: space-separated list of query reactions;
     if not provided, all reactions are considered).<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>GIMME FBA:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> </span>the objective
     function of GIMME method is to minimize the fluxes of lowly expressed
     reactions weighted by the deviations of reaction expression state from predefined
     threshold for low expression<span style='font-size:9.0pt;font-family:"Courier New"'>
     </span><span style='font-size:11.0pt;font-family:"Calibri","sans-serif";
     mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi'>(default
     12)</span>. The output is similar as FBA. The objective value is
     inconsistency score indicating the degree of disagreement between the gene
     expression data and the assumed objective function under specified
     required f<span style='font-family:"Calibri","sans-serif";mso-fareast-font-family:
     "DejaVu Sans"'>unctionalities</span><span style='mso-fareast-font-family:
     "Times New Roman"'>.<span style='mso-bidi-font-weight:bold'> (args: </span>space-separated
     list of query reactions; if not provided, all reactions are considered).<span
     style='mso-bidi-font-weight:bold'><o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>GIMME FVA:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> </span>the FVA version
     of GIMME method<span style='mso-fareast-font-family:"Times New Roman";
     mso-bidi-font-weight:bold'><o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>GIM3E:</span></b><span
     style='mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:
     bold'> predict reaction flux activities using metabolic constraints, the
     output is similar as FVA. (args: space-separated list of query reactions;
     if not provided, all reactions are considered).<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>SGNI:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> predict Strong
     Gene-Nutrient Interactions (sGNI) which uncover the dependence of gene
     essentiality on the presence/absence of nutrients given a growth medium of
     interest. The algorithm only needs a well-defined genome-scale model
     (args: biomass reaction; space-separated list of query genes; if not
     provided, all reactions are considered).<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>WGNI:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> predict Weak
     Gene-Nutrient Interactions (sGNI) using sampling approach in which a set
     of growth media is randomly sampled and normal FBA method are applied to
     predict gene knockout effects under each sampled medium (args: biomass
     reaction; space-separated list of query genes; if not provided, all
     reactions are considered).<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>DPAplot:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> <span style='mso-bidi-font-weight:
     bold'>find producibility plot<b> </b>for </span>Differential Producibility
     Analysis (DPA). The goal is to identify a set of genes that participate in
     the production/consumption of each metabolite. This is a binary matrix
     that links genes with metabolites on the basis of whether or not each gene
     is essential for production/consumption of each metabolite (no args).<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo5;tab-stops:list 36.0pt'><b style='mso-bidi-font-weight:
     normal'><span style='mso-fareast-font-family:"Times New Roman"'>DPAsig:</span></b><span
     style='mso-fareast-font-family:"Times New Roman"'> from producibility plot
     produced by dpaplot and by incorporating microarray gene expression data,
     here we are able to compute a value called ‘metabolite signal’ which
     reflects the metabolic state. The metabolite signal can be calculated as
     the median of associated gene expression ratios of its associated
     up/down-regulated genes (no args).<o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
margin-left:18.0pt'><span style='mso-fareast-font-family:"Times New Roman"'>Analysis
of iMAT method, Fast iMAT method, GIMME FBA/FVA and DPAsig needs expression
files which can be opened at <b style='mso-bidi-font-weight:normal'>File &gt;
Open expression</b>. Problem DPAsig also needs an output of DPAplot which can
be open at <b style='mso-bidi-font-weight:normal'>File &gt; Open DPAplot</b>.
Note that the analysis of iMAT method, Fast iMAT method, SGNI, WGNI, DPAplot
and DPAsig can only have one record in problem table. When you choose problem
SGNI and WGNI a tab titled ‘<b style='mso-bidi-font-weight:normal'>Nutrients</b>’
will be automatically popped up, in which you can specify the nutrients of
growth media to be tested (as ‘?nutrients’ in problem table) and present/absent
nutrients (as ‘$nutrients’ in problem table), then write into problem table. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Using
the toolbar&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Although the `Problem' table can be edited directly, JyMet provides a toolbar
which facilitates the definition of objective functions and argument lists in a
problem-specific format. To use the toolbar, select the analysis method in the <b>Analyse</b>
menu; the corresponding toolbar will appear. Then enter the objective function
and argument values wherever indicated. For Plot 3D, enter pairs of values
delimited by spaces, e.g. Reaction: r1 r2; From: -1.0 0; To: 1 2; Step: 0.1
0.2. If the second argument's value is equal to the first one, it may be
skipped. You can see SFM for specific problem type and the toolbar are
basically self-explanatory. <o:p></o:p></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'>The
`Reduce matrix' checkbox specifies whether the stoichiometry matrix has to be
reduced (see SFM for details). The `Comments' checkbox specifies whether
comments have to be included into the solution output. Once the toolbar fields
are filled, select <b>Solve &gt; Write problem</b>; the problem will be written
into the `Problem' table and can be further edited there. You can also clear
the problem table by clicking <b>Solve &gt; Clear problem.</b><o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Running
the solver&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Once the problem is defined, select <b>Solve &gt; Solve</b> to launch the
solver. While the solver is running, it can be suspended by selecting <b>Solve
&gt; Stop</b>. Simultaneous solution of problems is not allowed. You can choose
the different solver/algorithms in the `Solver' combo box on the toolbar ,
wherever they are applicable. For GLPK solver you can choose algorithms:
Simplex, Exact or MILP, and Gurobi is for Gurobi optimizer which can
automatically choose the simplex or MILP depends on problems (see SFM for
details). You can set parameter Dual&amp;Primal feasibility tolerance for
Gurobi solver by selecting <b style='mso-bidi-font-weight:normal'>Slove</b>
&gt; <b style='mso-bidi-font-weight:normal'>Tolerance</b>, tightening this
tolerance can produce smaller constraint violations (default: 1e-6, Min: 1e-9,
Max: 1e-2). Note that Gurobi solver needs to be pre-installed, it is free for
academics.<span style='mso-spacerun:yes'>  </span><o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Solution
output&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Once the solver exits, its output is written into a new tab which appears
rightmost in the model tab and whose title is the name of the analysis method.
The output format is method-specific, but the column names are
self-explanatory. Each record have corresponding record sub-tab and there is
tab ‘INFO.’ which gives information about analysis type and problem records and
corresponding SurreyFBA command line and running time spent. You can save the
output table through <b>File &gt; Save table</b><span style='mso-bidi-font-weight:
bold'>, or you can save the whole SurreyFBA output for current solved problem
by saving the temporary output file ‘out.tmp’ in directory ‘/jymet’ as another
file name.<span style='mso-spacerun:yes'>  </span><o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-fareast-font-family:"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<h2><a name="tth_sEc5"></a><a name="tth_sEc4"><span style='mso-bookmark:tth_sEc5'><span
style='mso-fareast-font-family:"Times New Roman"'>4</span></span></a><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>&nbsp;&nbsp;Visualisation<o:p></o:p></span></span></h2>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>JyMet can graphically represent models and solution results
in network diagrams or charts. <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><a name="OLE_LINK2"></a><a
name="OLE_LINK1"><span style='mso-bookmark:OLE_LINK2'><b><span
style='mso-fareast-font-family:"Times New Roman"'>Graph
visualisation&nbsp;&nbsp;</span></b></span></a></span><span style='mso-bookmark:
tth_sEc5'><span style='mso-bookmark:OLE_LINK1'><span style='mso-bookmark:OLE_LINK2'><span
style='mso-fareast-font-family:"Times New Roman"'> </span></span></span></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>This
functionality is intended for the visualisation of whole metabolic networks and
their subnetworks. Each vertex in a graph corresponds to an individual element:
gene, enzyme, reaction or metabolite and is connected with the vertices
representing its parents (if present) and children. <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>To draw a graph, select the cells containing the IDs of the
elements to be included. These cells can be selected in any table, including
`Problem' tables and those representing results. Note that an element is
included with its children (so, a gene is included with the enzymes encoded,
the reactions catalysed by these enzymes and the metabolites involved in their
equations), but not with its parents. The vertices are named by default with the
names of the genes, enzymes or metabolites or the reaction IDs. However, if
additional cells are selected, their contents are appended to the names,
delimited by semicolons (e.g. to visualise an elementary mode with the flux
rates, select the reaction IDs and the flux rates in the solution output
table). <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>The program uses two tools for graph drawing: the built-in
JGraph library, and the Graphviz tool, which needs to be pre-installed. To draw
a graph using JGraph, select <b>View &gt; Layout</b> and one of the radio buttons
specifying the layout methods; The graph drawing algorithm considers some
metabolites as `logical', e.g. every occurence of such a metabolite in a
reaction is represented by a separate vertex. This reduces the connectedness of
the networks and hence improves the readability of the diagrams. The list of
logical metabolites can be edited manually in the file `settings.py', which can
be found in the installation package. To draw it using Graphviz, select <b>View
&gt; Layout &gt; DOT</b>; in both cases, the graph will appear in a new tab. To
save it in a file, select <b>File &gt; Save table</b>; the file format will
depend on the file extension (e.g. png, bmp or jpg). <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><b><span
style='mso-fareast-font-family:"Times New Roman"'>Mapping results</span></b></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'><span
style='mso-spacerun:yes'>   </span>Apart from JGraph layouts, you can also
choose ‘Custom’ to load the user-built network graph to map the result on it.
Currently solution results of FBA, FVA, iMAT method, Fast iMAT method, GIMME
FBA, GIMME FVA and GIM3E can be mapped on graph, which are explained as
follows: <o:p></o:p></span></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>For FBA
     and GIMME FBA, the fluxes of selected reactions are mapped on reaction
     nodes and edges between reactions and metabolites indicating the
     metabolite fluxes which are calculated as its reaction flux multiplied by
     its coefficient.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>For FVA,
     the flux ranges of selected reactions are mapped on reaction nodes, such
     as 2.89~50.99 meaning from minimal flux 2.89 to maximal flux 50.99.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>For iMAT
     method, the ACT_LEVs of selected reactions are mapped on reaction nodes,
     such as 2f,-3r where ‘2f’ is ACT_LEV of 2 in forward direction and ‘-3r’
     is ACT_LEV of -3 in reverse direction if this is reversible reaction.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>For Fast
     iMAT method and GIMME FVA, the Activities of selected reactions are mapped
     on reaction nodes, where ‘f’ and ‘r’ are for active in forward and reverse
     direction, ‘-1’ is for inactive and ‘0’ is for undetermined activity.<o:p></o:p></span></span></li>
</ul>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>You can manipulate your graph by right clicking the mouse
where you can zoom in/out your graph, this can also be achieved by double
clicking your mouse (right for zoom out and left for zoom in); you can select
and search nodes or edges by all vertices/edges or by name; you also can
highlight the edges for FBA and GIMME FBA by set a set of flux thresholds to
classify the edge thickness, such as “1 10 50’ (separated by one space) which
gives 4 levels of edges thickness.<span style='mso-spacerun:yes'>  </span><o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>To draw it using Graphviz, select <b>View &gt; Layout &gt;
DOT</b>; in both cases, the graph will appear in a new tab. To save it in a
file, select <b>File &gt; Save table</b>; the file format will depend on the
file extension (e.g. png, bmp or jpg). <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>The graph drawing algorithm considers some metabolites as
`logical', e.g. every occurence of such a metabolite in a reaction is
represented by a separate vertex. This reduces the connectedness of the
networks and hence improves the readability of the diagrams. The list of
logical metabolites can be edited manually in the file `settings.py', which can
be found in the installation package. <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><b><span
style='mso-fareast-font-family:"Times New Roman"'>Building graph<span
style='mso-spacerun:yes'>   </span></span></b></span><span style='mso-bookmark:
tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:
bold'>To build a new network graph from scratch for a model or part of a model,
currently you can only use one of JGraph layouts to open a graph in JyMet then
adjust the graph by following operations: <o:p></o:p></span></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Drag the
     nodes or edges to desired positions. <o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Press
     Control key at the same time drag the nodes or edges to copy them.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Move
     mouse over the nodes or edges then right click to select ‘Delete’ to
     delete them.<o:p></o:p></span></span></li>
 <li class=MsoNormalCxSpMiddle style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
     auto;mso-add-space:auto;mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span
     style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Move
     mouse to a start node, keep pressing left button of mouse and move it to
     an end node to get an edge between 2 nodes.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Double
     click the text in nodes/edges to edit them. <o:p></o:p></span></span></li>
</ul>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'>After that, you can save your
graph as a text file by selecting <b>File&gt;Save graph</b>. This file
describing the nodes and edges information related to the graph. You can also
modify your graph by editing its graph text file. The format of graph file is
as follows. The first line gives graph name, and then each line describes a
node/edge by the </span>tab-delimited</span><span style='mso-bookmark:tth_sEc5'><span
style='mso-fareast-font-family:"Times New Roman";mso-bidi-font-weight:bold'>
columns. For nodes the meaning of 8 columns from left to right are as follows: <o:p></o:p></span></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>With
     ‘Node’ indicating a node.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node id,
     a metabolite can have many copies with different ids related to different
     reactions.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node
     name presented in the graph.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node
     position in X axis, the distance to the left side of graph.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node
     position in Y axis, the distance to the up side of graph.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Height
     of node outline.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Width of
     node outline.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>A set of
     features describing appearance of nodes which are self-explained<span
     style='mso-bidi-font-weight:bold'>. <o:p></o:p></span></span></span></li>
</ul>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'>For edges the meaning of 6 columns
from left to right are as follows:<o:p></o:p></span></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>With
     ‘Edge’ indicating an edge.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Edge id,
     normally a number produced by JyMet. <o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Edge
     information between 2 nodes such as stoichiometry, flux.<span
     style='mso-spacerun:yes'>  </span><o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node id
     the edge starts from.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Node id
     the edge points to.<o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>A set of
     features describing appearance of edges which are self-explained. <o:p></o:p></span></span></li>
</ul>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'>Edge types: the normal edge is
solid line with triangle arrow; the edge from an activator is dashed line with
triangle arrow; the edge from an inhibitor is dashed line with oval arrow.<o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman";mso-bidi-font-weight:bold'><span
style='mso-spacerun:yes'> </span><o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><b><span
style='mso-fareast-font-family:"Times New Roman"'>Charts&nbsp;&nbsp;</span></b></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>
One or more solution outputs can be visualised in a chart, whose type depends
on the analysis method: <o:p></o:p></span></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>FBA
     results, iMAT method, WGNI, DPAplot, DPAsig, elementary modes, minimal
     substrate and product sets are shown in bar charts. <o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>FVA and
     Fast iMAT method results are shown in box and whisker charts. <o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Plot
     results are shown in line plots. <o:p></o:p></span></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo6;tab-stops:list 36.0pt'><span style='mso-bookmark:
     tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>Plot 3D
     results are shown in grayscale block plots. <o:p></o:p></span></span></li>
</ul>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>The first three types of the charts listed above can cover
multiple results from different output tables (e.g. in a line plot, they are
shown as different curves). The chart legends are extracted from the output
comments, if they are selected. <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>To visualise one or more results in a chart, select the
corresponding rows in the output tables and select <b>View &gt; Plot</b>; the
chart will appear in a new tab. For DPAplot and DPAsig output tables you can
select specific row and column to plot. To edit the chart, click the right
mouse button in it and select <b>Properties</b> in the popup menu. To save the
chart in a png file, select either <b>File &gt; Save table</b>, or <b>Save</b>
in the context menu. <o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'><o:p>&nbsp;</o:p></span></span></p>

<h2><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>5&nbsp;&nbsp;QSSPN<o:p></o:p></span></span></h2>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>The menu QSSPN contains the operations implemented for QSSPN
(Quasi-Steady State Petri Nets) software (see website at </span></span><span
style='mso-bookmark:tth_sEc5'><tt><u><span style='font-size:10.0pt;color:blue'>http://sysbio3.fhms.surrey.ac.uk/qsspn/index.html</span></u></tt></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>).
To run the QSSPN, firstly you need to open a </span></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:SimSun'>GSMN
model of </span>SurreyFBA reaction table file</span><span style='mso-bookmark:
tth_sEc5'><span style='mso-fareast-font-family:SimSun'> (<b style='mso-bidi-font-weight:
normal'>File</b> &gt; <b style='mso-bidi-font-weight:normal'>Open model</b>) which
will be used by QSSPN model, then open</span> </span><span style='mso-bookmark:
tth_sEc5'><span style='mso-fareast-font-family:SimSun'>corresponding </span></span><span
style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:"Times New Roman"'>QSSPN
model (<b style='mso-bidi-font-weight:normal'>QSSPN<span
style='mso-spacerun:yes'>  </span>&gt; Open QSSPN</b>) or import a SPEPT model
file built by Snoopy software (<b style='mso-bidi-font-weight:normal'>QSSPN<span
style='mso-spacerun:yes'>  </span>&gt; Import SPEPT</b>), then you have to load
a QSSPN control file (<b style='mso-bidi-font-weight:normal'>QSSPN<span
style='mso-spacerun:yes'>  </span>&gt; Load control</b>). To run QSSPN software
you just click <b style='mso-bidi-font-weight:normal'>QSSPN<span
style='mso-spacerun:yes'>  </span>&gt; Run</b> and stop it by <b
style='mso-bidi-font-weight:normal'>QSSPN<span style='mso-spacerun:yes'> 
</span>&gt; Stop. <o:p></o:p></b></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'>You can edit QSSPN model file and control file by modifying
the field values and also you can insert or delete rows through <b
style='mso-bidi-font-weight:normal'>Edit &gt; Insert rows</b> and <b
style='mso-bidi-font-weight:normal'>Edit &gt; Delete rows.<span
style='mso-spacerun:yes'>  </span></b>After editing<b style='mso-bidi-font-weight:
normal'> </b>you can save the QSSPN model and Control parameters into files in
QSSPN formats. When you get output of QSSPN, you can use <b style='mso-bidi-font-weight:
normal'>QSSPN<span style='mso-spacerun:yes'>  </span>&gt; Plot trajectory</b>
to draw a plot showing multiple trajectories by selecting the target node
columns and time span. You can even open a saved output file (<b
style='mso-bidi-font-weight:normal'>QSSPN &gt; Open output</b>) to view and
plot them.<o:p></o:p></span></span></p>

<p class=MsoNormal><span style='mso-bookmark:tth_sEc5'><span style='mso-fareast-font-family:
"Times New Roman"'><o:p>&nbsp;</o:p></span></span></p>

<span style='mso-bookmark:tth_sEc5'></span>

<h2><span style='mso-fareast-font-family:"Times New Roman"'>6&nbsp;&nbsp;Final
remarks<o:p></o:p></span></h2>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>System
requirements&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
JyMet is written in Jython (a nice hybrid of Python and Java), and is intended
to be platform-independent, only requiring a Java virtual machine to be
installed. So far it has been successfully tested on various versions of Linux,
Windows and Mac. <span style='mso-bidi-font-weight:bold'>Two versions of
software are provided, one is called sfba-glpk which only uses GLPK library,
and the other is called sfba-grb which uses GLPK and Gurobi libraries, note
that before use this version you have to install licensed Gurobi solver.</span><span
style='mso-spacerun:yes'>  </span><o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Reporting
bugs&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
Please report bugs to h.wu@surrey.ac.uk. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>Copyright&nbsp;&nbsp;</span></b><span
style='mso-fareast-font-family:"Times New Roman"'>© 2013 Albert Gevorgyan and
Andrzej Kierzek and Huihai Wu. This program is free software; you may
redistribute it under the terms of the GNU General Public License. This program
has absolutely no warranty. <o:p></o:p></span></p>

<p class=MsoNormal><b><span style='mso-fareast-font-family:"Times New Roman"'>See
also&nbsp;&nbsp;</span></b><span style='mso-fareast-font-family:"Times New Roman"'>
<o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>SurreyFBA manual, which should be distributed with this
     program <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of the underlying LP solver GLPK and its
     C API (<a href="http://www.gnu.org/software/glpk/glpk.html"><tt><span
     style='font-size:10.0pt'>http://www.gnu.org/software/glpk/glpk.html</span></tt></a>)
     <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentaion of Gurobi slover and its installation
     information (</span><tt><u><span style='font-size:10.0pt;color:blue'>http://www.gurobi.com/</span></u></tt><span
     style='mso-fareast-font-family:"Times New Roman"'>)<o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of Jython (<a href="http://jython.org"><tt><span
     style='font-size:10.0pt'>http://jython.org</span></tt></a>) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of JFreeChart library, used for drawing
     charts (<a href="http://www.jfree.org/jfreechart"><tt><span
     style='font-size:10.0pt'>http://www.jfree.org/jfreechart</span></tt></a>) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of JGraph library, used for drawing
     graphs (<a href="http://www.jgraph.com"><tt><span style='font-size:10.0pt'>http://www.jgraph.com</span></tt></a>)
     <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of Graphviz software, used as an
     alternative for drawing graphs (<a href="http://www.graphviz.org"><tt><span
     style='font-size:10.0pt'>http://www.graphviz.org</span></tt></a>) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of pydot library: a Python interface to
     Graphviz (<a href="http://dkbza.org/pydot/pydot.html"><tt><span
     style='font-size:10.0pt'>http://dkbza.org/pydot/pydot.html</span></tt></a>)
     <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>Documentation of SBML (<a href="http://sbml.org"><tt><span
     style='font-size:10.0pt'>http://sbml.org</span></tt></a>) <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo7;tab-stops:list 36.0pt'><span style='mso-fareast-font-family:
     "Times New Roman"'>GNU General Public License, which should be distributed
     with this program (<a href="http://www.gnu.org/licenses/gpl.txt"><tt><span
     style='font-size:10.0pt'>http://www.gnu.org/licenses/gpl.txt</span></tt></a>)
     <o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='margin-bottom:12.0pt'><span style='mso-fareast-font-family:
"Times New Roman"'><o:p>&nbsp;</o:p></span></p>

<div class=MsoNormal align=center style='text-align:center'><span
style='mso-fareast-font-family:"Times New Roman"'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal><span style='font-size:10.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_3" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: 2 Colour Surrey Logo"
 style='width:100.5pt;height:29.25pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Manual_files/image001.jpg" o:title=" 2 Colour Surrey Logo"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=134 height=39
src="Manual_files/image002.jpg"
alt="Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: Description: 2 Colour Surrey Logo"
v:shapes="Picture_x0020_3"><![endif]></span><span style='mso-fareast-font-family:
"Times New Roman"'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:10.0pt'>Faculty of Health and Medical
Sciences<br>
University of Surrey<span style='mso-spacerun:yes'>                          
</span></span><span style='mso-fareast-font-family:"Times New Roman"'><o:p></o:p></span></p>

</div>

</body>

</html>
